backend/src/notes/notes.service.ts

```diff
@@ -94,6 +94,12 @@ export class NotesService {
     // Check if new note doesn't violate application constraints
     if (alias) {
       this.checkNoteIdOrAlias(alias);
```

Comment:
since we have multiple check methods. We should rename this so it is more obvious what is checked.